README.md

```diff
@@ -89,17 +89,3 @@ console.log(fieldPrefix);
 > Note: Sample model definition is available in [data/model.json](data/model.json)
 
 
-## Contributing
```

Comment:
We do not want to remove this section `Contributing` 

can you revert this.

Reply:
okay, will do!